ci/builders/README.md

```diff
@@ -374,9 +374,126 @@ be relative to the checkout directory.
 
 ### Global Tests
 
-Global tests follow the same format as build tests but defined at the top
-level. The main difference from local tests is that global tests have access
-to the sub-build artifacts.
+Tests in this section run on a separate bot as an independent sub-builds.
+As opposed to tests running within builds, global tests have access to the
+the outputs of all the builds running in the same orchestrator build. A
+use case for global tests is to run flutter/framework tests using the
+artifacts generated by an specific engine build.
+
+Global tests currently support two different scenarios:
+
+* flutter/flutter tests with [tester](https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/engine_v2/tester.py)
+  recipe. This workflow checks out flutter/flutter to run any of the existing
+  sharded tests using the engine artifacts archived to GCS.
+* complicated engine tests that require the outputs from multiple subbuilds
+  with [tester_engine](https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/engine_v2/tester_engine.py).
+  This workflow checks out [flutter/engine] and operates over the dependencies passed to it using cas.
+
+Note: the supported scenarios can be later extended to support running devicelab tests although a
+smart scheduler is a prerequisite for it to be scalable.
+
+Framework test example:
+
+```json
+{
+   "tests": [
+      {
+        "name": "web-tests-1",
+        "shard": "web_tests",
+        "subshard": "1",
+        "test_dependencies": [
+          {
+            "dependency": "chrome_and_driver",
+            "version": "version:111.0a"
+          }
+        ]
+      }
+    ]
+}
+```
+
+The property's description is as follows:
+
+* **name** the name that will be assigned to the sub-build.
+* **shard** the flutter/flutter shard test to run.
+* **subshard** one of the accepted subshard values for shard.
+* **test_dependencies** a list of [dependencies](https://flutter.googlesource.com/recipes/+/refs/heads/main/recipe_modules/flutter_deps/api.py#75)
+  required for the test to run.
+
+Engine test example:
+
+```json
+{
+  "tests": [
```

Comment:
Okay so are the fields that you can provide in the test object different based on the test recipe used? For example tester.py and tester_engine.py? It appears to be that way.

Reply:
That's correct each recipe expect different properties.